### DISCUSSION TITLE
i cant play prodigy becus my school blocks it

### DISCUSSION THREAD

**Discussion by ghost** [Prodigy-Hacking/ProdigyMathGameHacking#911] [(B) Bug Discussion]
pls help

**Comment by ghost** [comment]
just like tell meh how to still do it

**Comment by ghost** [comment]
allso i tried is.gd it did not work???

**Reply by ghost** [reply]
pls help @pp]

**Reply by ghost** [reply]
oooooooooooooooooooooppppps

**Reply by NX721** [reply]
if it's an extension we can't help

**Reply by ghost** [reply]
nope it prodigy math game

**Comment by ghost** [comment]
@BETPRODIGYHACKER I can help is it blocked by an extension (eg. Blocksi)

**Reply by ghost** [reply]
just tell how they blocked it

**Comment by LeoBadeaux** [comment]
Fix it yourself, after all you are the "BEST PRODIGY HACKER"

**Reply by ghost** [reply]
allso if i download aprivate browser it just block download and allso they block extinsions and no they dont block prodigy with a extinsion

**Comment by ghost** [comment]
no he is the bet prodigy hacker

**Reply by ghost** [reply]
its looks like this when im blocked Web Page Blocked
The web page you are trying to visit has been blocked in accordance with company policy. Please contact your system administrator if you believe this is an error.

User: lwsd\s-nyalla

URL: www.prodigygame.com/

Category: District Blacklist

**Reply by ghost** [reply]
ok

**Reply by NX721** [reply]
Use a vpn

**Reply by ghost** [reply]
uuum  i do not know how

**Reply by ghost** [reply]
can u teach me???

**Comment by ghost** [comment]
#928
